packages/ui/src/camel-utils/camel-to-tile.adapter.ts

```diff
@@ -30,6 +31,7 @@ export const camelComponentToTile = (componentDef: ICamelComponentDefinition): I
     tags,
     provider,
     version,
+    nodeIconType: NodeIconType.Component,
```

Comment:
@tplevko, could be possible to use the tile type to determine the `NodeIconType` instead? this way we wouldn't need to add a new property here.

Reply:
sure, I used the title type instead of the `nodeIconType`, which I removed. Thanks for suggestion.